tests/bluetooth/tester/src/gap.c

```diff
@@ -1176,6 +1190,33 @@ static void set_oob_legacy_data(const uint8_t *data, uint16_t len)
 		   CONTROLLER_INDEX, BTP_STATUS_SUCCESS);
 }
 
+static void set_filter_list(const uint8_t *data, uint16_t len)
+{
+	const struct gap_set_filter_list *cmd = (const void *) data;
+	uint8_t status = BTP_STATUS_FAILED;
```

Comment:
Why not set the status in the error branches, same way as you do in the `connect()` function?

Reply:
updated, status is now set in error branches